app/src/test/java/by/academy/jc/ht/zhabko/AverageCounterThreadsTest.java

```diff
@@ -0,0 +1,47 @@
+package by.academy.jc.ht.zhabko;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+public class AverageCounterThreadsTest {
```

Comment:
Use https://docs.oracle.com/javase/8/docs/api/java/util/stream/IntStream.html#average-- to check yourself